src/Sinch/Faxes/Fax.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Sinch.Faxes
+{
+    
+        /// <summary>
+        /// Fax object, see https://developers.sinch.com/docs/fax for more information
+        /// </summary>
+        public class Fax
+        {
+            [ReadOnly(true)]
```

Comment:
The request for fax can be json or multipart, but response is json only. `System.ComponentModel.DataAnnotations` doesn't work with `System.Text.Json`.

Reply:
tahts true, maybe this method is to much but i do want to have a few short cut methods where you dont need to care. like sens(to, file) where they dont simply care about anything more. This is especially powerful for example in conversation where we did a misstake and thought everybody wanted to have contacts, you could have a send method that takes a to and body argument to send an SMS wich is probably 80% of the traffic, no message type, no contact

makes the api look super simple,

Reply:
didnt know annotations didnt work, on json. but it would work if someone set the wrong before serialization here right? we dotn return wrong stuff, want to prevent developer from doing it and help them with format.

actually we do have binary on a  get as well, and webhooks are multipart NOT json. but im not sure what you are asking here

Reply:
Data annotations can work with `Newtonsoft.Json`.
But currently they do nothing. so they could be removed.

Reply:
What I want is to throw and Exception to the developer before its sent to server, and give them a hint what we expect. in this case a e164 formatted phonenumber. Maybe we should have a common validator for phonenumbers (and prob use google phonelib for it)

Reply:
We just head a discussion yesterday with @JPPortier in which his attempt to validate e164. It worked for some APIs but failed with Voice, so use of api was denied by sdk when server was accepting the value just fine(or it was that server returned the wrong e164 number, @JPPortier can remind the details).

What I'm leading to - is that I don't want to use client side validation - if server changes or not consistent - we can deny user use of api unintentionally. And, anyway, api will handle the error. How much we are adding problem to api throughput? I can't argue here without real numbers. Maybe e164 makes sense in the end, but practice shows it's not that consistent - so api itself should be the point of truth. 

What can we validate? Obvious error - empty string for example. I even have doubts about use of `required`.

Reply:
> Maybe we should have a common validator for phonenumbers (and prob use google phonelib for it)

Additionally, less third-party dependencies is better, ideally zero(we are here now)

Reply:
> We just head a discussion yesterday with @JPPortier in which his attempt to validate e164. It worked for some APIs but failed with Voice, so use of api was denied by sdk when server was accepting the value just fine(or it was that server returned the wrong e164 number, @JPPortier can remind the details).

Exact
With Voice API, when you are sending a "proper" E164 number (+33628254417) but when Voice API is sending back some responses about the call (WebHooks, ...) then the number is no longer valid (`+` sign removed)
So you do not receive what you sent and finally it is ourselves (Voice product) which make impossible to validate  and refuse invalid data 

I wish not having to answer to an end user that his application using our SDK is failing because SDK cannot accept a response from our own servers because received payload is invalid and different from his request 😨 
I'm still thinking this could be an added value to be able to do it from client side it but for phone numbers it seems to be a dead end (for DTMF format, currently, the client side validation is still active)

Reply:
Hey, i understand that. But that one api returns invalid data should not prevent another product for decorating its property for proper data if you always do return that. That voice bug is a famous old one. and yes DTMF etc would make it impossible to mark that request in voice as a e164 number since its not i guess.

Reply:
and sure i get that no dependices are great. but also dealing with phonenumbers is complex so its also a great help for a developer. So either  dependency or include the .net implementation in our build so no dependents